docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+---
+layout: page
+title: Email Components
+description: Use components as building blocks as you design and build new emails.
+---
+<section class="stacks-section">
+    {% header {element: 'h2', text: 'Typography'} %}
+    <p class="stacks-copy">
+        It's safe and accessible to use semantic HTML tags like <code class="stacks-code">&lt;h&gt;</code>, <code class="stacks-code">&lt;p&gt;</code>, and <code class="stacks-code">&lt;ul&gt;</code> for text in email just as we do for the web. The main difference in email is that <a href="/email/faq#whydowehavetowritecssinline">CSS should be written inline</a> to specify intended styles (like the color of an <code class="stacks-code">&lt;a href&gt;</code> tag) and zero out unintended defaults (like the margin around a <code class="stacks-code">&lt;p&gt;</code> tag)
+    </p>
+    <p class="stacks-copy">
+        This page provides some common typography patterns for email, but anything on <a href="/base/typography">the main Stacks typography page</a> is fair game too (as long as CSS is written inline, of course).
+    </p>
+    {% header {element: 'h3', text: 'Paragraphs'} %}
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<p style="margin: 0 0 15px 0; font-size: 17px; line-height: 23px;">A lead paragraph with slightly larger text lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras nisi magna, mollis luctus sapien varius, mattis ornare est.</p>
+<p style="margin: 0 0 15px 0;">A regular paragraph with <strong>strong</strong> and <em>italic</em> text, consectetur adipiscing elit. Cras nisi magna, mollis luctus sapien varius, mattis ornare est. Sed non tincidunt erat, ut commodo augue.</p>
+<p style="margin: 0;">A paragraph with no margin (useful for the last paragraph in a section).</p>
+{% endhighlight %}
+        <div class="stacks-preview--example" style="font-family: arial, sans-serif; font-size: 15px; line-height: 19px; color: #54595F; text-align: left;">
+            <p style="margin: 0 0 15px 0; font-size: 17px; line-height: 23px;">A lead paragraph with slightly larger text lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras nisi magna, mollis luctus sapien varius, mattis ornare est.</p>
+            <p style="margin: 0 0 15px 0;">A regular paragraph with <strong>strong</strong> and <em>italic</em> text, consectetur adipiscing elit. Cras nisi magna, mollis luctus sapien varius, mattis ornare est. Sed non tincidunt erat, ut commodo augue.</p>
+            <p style="margin: 0;">A paragraph with no margin (useful for the last paragraph in a section).</p>
+        </div>
+    </div>
+    {% header {element: 'h3', text: 'Headlines'} %}
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<h1 style="font-weight: bold; font-size: 27px; line-height: 27px; color: #242629; margin: 0 0 15px 0;">I'm a headline</h1>
```

Comment:
The color here is `#242629`, aka `@black-800`. Shouldn't we use `@fc-dark` (`@black-900`) or `@fc-medium` (`@black-700`) instead? I'd opt for `@fc-dark`.

Also why `15px` bottom margin instead of `16px`?

Reply:
Yep, agree on using `@fc-dark` for headings, `@fc-medium` for regular text, and `@fc-light` for light text (eg. footer). Will update.

RE margins, hybrid math is easiest when the widths and horizontal spacing in multiples of 5. I simply carried this over to vertical spacing as well so everything's symmetrical. No hard rules here, though. Maybe we should document somehow?